posthog/helpers/session_recording.py

```diff
@@ -17,6 +17,20 @@
 WindowId = Optional[str]
 
 
+@dataclasses.dataclass
+class RecordingEventForObjectStorage:
```

Comment:
Makes sense, I updated this to be `ChunkedRecordingEvent` which I think describes it properly. I wasn't using `RecordingEvent` because it's pretty overloaded right now.